keps/sig-api-machinery/2211-omit-managed-fields-server-side/kep.yaml

```diff
@@ -0,0 +1,25 @@
+title: Omit managedFields on the server-side
+kep-number: "2211"
+authors: ['@knight42']
+owning-sig: sig-api-machinery
+participating-sigs: [sig-cli]
+reviewers: ['@apelisse', '@kwiesmueller']
+approvers: ['@wojtek-t', '@lavalamp']
+prr-approvers: ['@wojtek-t']
```

Comment:
If you want to target alpha in 1.21, you need to fill PRR and get approval before feature freeze.